elliottlib/cli/__main__.py

```diff
@@ -94,10 +94,14 @@
               help='Remove found bugs from ADVISORY')
 @use_default_advisory_option
 @click.option("--id", metavar='BUGID', default=[],
-              multiple=True, required=True,
+              multiple=True, required=False,
               help="Bugzilla IDs to add.")
+@click.option("--all", "remove_all",
+              required=False,
+              default=False, is_flag=True,
+              help="AUTO mode, remove all bugs attached to ADVISORY")
```

Comment:
`AUTO mode` sound vague to me.
would like to rephrase to something like `Remove all attached bugs from ADVISORY`